dokka-subprojects/plugin-base/src/test/kotlin/model/InheritorsTest.kt

```diff
@@ -425,4 +433,89 @@ class InheritorsTest : AbstractModelTest("/src/main/kotlin/inheritors/Test.kt",
             }
         }
     }
+
+    @Test
+    fun `java nested classes should not be inherited`() {
+        testInline(
+            """
+            |/src/main/kotlin/sample/ParentInKotlin.kt
+            |package sample
+            |class Child: JavaParent()
+            |class KotlinChild: KotlinParent() 
+            |open class KotlinParent {
+            | class NestedKotlinParent
+            | inner class InnerClass
+            |}
```

Comment:
Are `KotlinChild` and `KotlinParent` needed in this test? If no, can they be removed? Otherwise it's unclear if they are a prerequisite to reproducing the problem in some weird way

Reply:
It is not needed.